src/atom_finder/classifier/operator-precedence.clj

```diff
@@ -51,16 +51,16 @@
 (defn specific-confusing-operator-combination?
   "if the combination of groups of operators exists in this set, then the combination is confusing"
   [combination]
-  (some #(= % combination) (map sort [[:de_incr :pointer] [:multiply :add] [:arith_unary :add] [:arith_unary :multiply]
-                                      [:and :add] [:and :multiply] [:and :arith_unary] [:or :add]
-                                      [:or :multiply] [:or :arith_unary] [:or :and] [:not :add]
-                                      [:not :multiply] [:not :arith_unary] [:not :and] [:not :or]
-                                      ;;[:compare :and] [:compare :or] // Underclassify
-                                      [:compare :not] [:compare :compare] [:pointer :add] [:cond :arith_unary]
-                                      [:cond :and] [:cond :or] [:cond :not] [:cond :compare]
-                                      [:cond :cond] [:non-asso :add] [:non-asso :multiply] [:non-asso :arith_unary]
-                                      [:non-asso :and] [:non-asso :or] [:non-asso :not] [:non-asso :non-asso]
-                                      [:field_ref :pointer]])))
+  (some #(= % combination) (into #{} (map sort [[:de_incr :pointer] [:multiply :add] [:arith_unary :add] [:arith_unary :multiply]
```

Comment:
thanks for doing this, but it actually misses the main benefit of what I was proposing. the construction of the set `(into #{}` happens every time `specific-confusing-operator-combination?` is run, which is lame, additionally, `(some #(= % combination)` has to loop over every element of the set, where as if you did a set-lookup it's constant-time. So, if you wanted to make this better you could:

1) put `(into #{} (map ...` into a `def` variable outside the function, so it's only created one time ever
2) don't use `(some XXX set)` on a set, instead just do `(set XXX)`

That said, if you're sick of this, we can commit the code now, up to you, just let me know.